services/vision/obstaclesdepth/obstacles_depth.go

```diff
@@ -117,12 +117,15 @@ func (o *obsDepth) buildObsDepth(logger logging.Logger) func(
 
 // buildObsDepthNoIntrinsics will return the median depth in the depth map as a Geometry point.
 func (o *obsDepth) obsDepthNoIntrinsics(ctx context.Context, src camera.VideoSource) ([]*vision.Object, error) {
-	pic, release, err := camera.ReadImage(ctx, src)
+	imgBytes, mimeType, err := src.Image(ctx, utils.MimeTypeRawDepth, nil)
```

Comment:
I think you can just leave it blank here as well for the MimeType request

Reply:
Leaving blank causes re-encoding issues ([take a look](https://github.com/viamrobotics/rdk/pull/4487/commits/11b1d7d567c22fb428f3245a46a53f0b5fbff1a1))